test/chat_api/notes_test.exs

```diff
@@ -0,0 +1,64 @@
+defmodule ChatApi.NotesTest do
+  use ChatApi.DataCase
+
+  alias ChatApi.Notes
+
+  describe "notes" do
+    alias ChatApi.Notes.Note
+
+    @valid_attrs %{body: "some body"}
+    @update_attrs %{body: "some updated body"}
+    @invalid_attrs %{body: nil}
+
+    def note_fixture(attrs \\ %{}) do
```

Comment:
We have moved to factories from fixtures. Have a look on factory.ex you will need to create something like 

```
def note_factory do 
    %ChatApi.Notes.Note{ 
          account: build(:account),
          author: build(:user),
          customer: build(:customer)
     }
end
```
You will need to import ChatApi.Factory and then within your tests you will have access to ex_machinas comands
insert,insert_pair,insert_list, build, params_for, params_with_assocs for more info you can see [ex_machina docs](https://hexdocs.pm/ex_machina/readme.html) or peek into accounts_test.ex (or other tests) to see it in use.

Let me know if I can be of any help.

Reply:
i think `author` is fine, since it's a bit more descriptive than `user` (Intercom uses `author` as well in their notes API: https://developers.intercom.com/intercom-api-reference/reference#note-model -- not that we should blindly copy them, but i think it's a reasonable name 😄 )

Reply:
i think `author` is fine, since it's a bit more descriptive than `user` (Intercom uses `author` as well in their notes API: https://developers.intercom.com/intercom-api-reference/reference#note-model -- not that we should blindly copy them, but i think it's a reasonable name 😄 )

Reply:
Thanks @flmel , the factories are much nicer to use than fixtures! I gave it a shot - let me know what you think?